### PR TITLE
Refactoring: follow happy path: removed else blocks that are not needed

### PR DESCRIPTION
**Pull request by tisnik**
## Description

Follow happy path: removed else blocks that are not needed

## Type of change

- [x] Refactor
- [ ] New feature
- [ ] Bug fix
- [ ] CVE fix
- [ ] Optimization
- [ ] Documentation Update
- [ ] Configuration Update
- [ ] Bump-up dependent library
- [ ] Bump-up library or tool used for development (does not change the final image)

### CODE REVIEW

**Comment by tisnik** [comment]
/override "Red Hat Konflux / lightspeed-service-on-pull-request"

File: ols/app/endpoints/health.py [hunk 1]
```diff
@@ -60,8 +58,7 @@ def readiness_probe_get_method() -> ReadinessResponse:
         return ReadinessResponse(ready=False, reason="index is not ready")
     if not llm_is_ready():
         return ReadinessResponse(ready=False, reason="LLM is not ready")
-    else:
-        return ReadinessResponse(ready=True, reason="service is ready")
+    return ReadinessResponse(ready=True, reason="service is ready")
```
Comment (syedriko) [inline ols/app/endpoints/health.py:61]:
```
return ReadinessResponse(
    ready=llm_is_ready(), 
    reason="service is ready" if llm_is_ready() else "LLM is not ready"
)
```

**Pull request merged**